search-service/src/test/kotlin/com/egm/stellio/search/web/TemporalEntityHandlerTests.kt

```diff
@@ -816,4 +818,187 @@ class TemporalEntityHandlerTests {
         assertEquals(null, temporalQuery.time)
         assertEquals(null, temporalQuery.timerel)
     }
+    @Test
+    fun `query subscriptions should return 200 with prev link header if exists`() {
```

Comment:
in such tests, you should also check the link content (like is done for instance in https://github.com/stellio-hub/stellio-context-broker/blob/develop/subscription-service/src/test/kotlin/com/egm/stellio/subscription/web/SubscriptionHandlerTests.kt#L287). But nothing else (for instance, the response body is already checked in other tests).